io/js/src/main/scala/fs2/io/Stream.scala

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Can you rename this file to something else? It will conflict with folks using editor quick open features for Stream.scala

Reply:
Good point, done. Thanks!